### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by Atefeh-Behroozi**

### CODE REVIEW

**Comment by kralle333** [comment]
@Atefeh-Behroozi I see the linter pointed out some issues, can you have a look at them before i review?
Also it would be very nice with some description on the PR so I have any idea what the change is about.
If the code is for frontend code, I really like when people add screenshots of what the changes entail.

**Comment by Atefeh-Behroozi** [comment]
Thank you @kralle333 for the feedback. I've fixed my issue the linter.
For branches that I made and send here without any comment, I have to say, I was in the middle of styling and We have to go to the dev and fix some conflict and merge it, So I've just committed my code. Finish project page is the whole codes that I wrote for frontend project page.  The last one, fix conflict, Also was for dev branch.

**Comment by kralle333** [comment]
> Thank you @kralle333 for the feedback. I've fixed my issue the linter. For branches that I made and send here without any comment, I have to say, I was in the middle of styling and We have to go to the dev and fix some conflict and merge it, So I've just committed my code. Finish project page is the whole codes that I wrote for frontend project page. The last one, fix conflict, Also was for dev branch.

So..is it ready for me to review now? :)

**Comment by Atefeh-Behroozi** [comment]
Yes Yes. Now It is ready for review :)

**Comment by kralle333** [comment]
You have a conflict with the branch you want to merge into and I also I asked to have a description of your changes, which you have not given me :)

**Comment by Atefeh-Behroozi** [comment]
I fix conflict now. I worked on ProjectPage folder. I designed frontend page , that name is projectPage and inside of the page there are some projects with name and detail. Also I added a modal, when click on button, modal is open and you can create a new project. I'm not sure about my description, If isn't enough I can explain more :)

File: frontend/src/pages/projectPage/ProjectList.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import ProjectModal from './ProjectModal';
+import TaskList from './TaskList';
+import './ProjectList.css';
+import image1 from '../../assets/images/Rectangle 2996.jpg';
+import image2 from '../../assets/images/Rectangle 2997.jpg';
+import image3 from '../../assets/images/Rectangle 2998.jpg';
+import image4 from '../../assets/images/Rectangle 2999.jpg';
```
Comment (kralle333) [inline frontend/src/pages/projectPage/ProjectList.tsx:8]: how are you planning on managing these assets in the future? Is the user supposed to upload their own project images or choose from these ones?

File: frontend/src/pages/projectPage/TaskItem.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import Assigned from "../../assets/icons/assigned task.png";
+import Circle from "../../assets/icons/Danger Circle.png";
+
+interface Project {
+  id: number;
+  title: string;
+  thumbnail_link: string;
+  date_of_creation: string;
+  amount_of_tasks: number;
+}
```
Comment (kralle333) [inline frontend/src/pages/projectPage/TaskItem.tsx]: duplicate interface also found in TaskList. Did you sync with develop? I think someone already made a Project class

**Review by kralle333 — CHANGES_REQUESTED** [review]
Good work. I think you need to merge develop into your branch and then use the Project class that has been made by your team mate. 

After that I will approve :)

**Comment by Atefeh-Behroozi** [comment]
Thank you for your feedback, I used from interface project --> types and deleted my interfaces.

File: frontend/src/pages/projectPage/ProjectList.tsx (hunk 1 above)
Comment (Atefeh-Behroozi) [reply frontend/src/pages/projectPage/ProjectList.tsx:8]: I'm actually not sure about this, I have to speak with my team mates :)

**Review by kralle333 — APPROVED** [review]

**Pull request merged**